src/main/java/frc/team2412/robot/Controls.java

```diff
@@ -128,14 +128,16 @@ public void bindShooterControls() {
             BooleanSupplier b = driveController.getDPadButton(Direction.UP)::get;
 
             driveController.getDPadButton(Direction.DOWN).whenPressed(
-                    new ShooterHoodRPMCommand(subsystems.shooterSubsystem, 1000, 35).withInterrupt(b));
+                    new ShooterHoodRPMCommand(subsystems.shooterSubsystem, 1400, 35).withInterrupt(b)
+                            .alongWith(new InstantCommand(() -> subsystems.shooterSubsystem.setTurretAngle(-90))));
```

Comment:
I wasn't programming during comp, what were the reasons for:

1. Making the interrupt only affect the hood and flywheel command (Less relevant because instant commands finish instantly anyways)
2. Switching from `ShooterTurretSetAngleCommand` to an `InstantCommand`